cloud_images/centos7/install_prereqs.sh

```diff
@@ -21,7 +21,7 @@ sysctl -w net.ipv6.conf.all.disable_ipv6=1
 sysctl -w net.ipv6.conf.default.disable_ipv6=1
 
 echo ">>> Installing DC/OS dependencies and essential packages"
-yum install --assumeyes --tolerant perl tar xz unzip curl bind-utils net-tools ipset libtool-ltdl rsync
+yum -y --tolerant install perl tar xz unzip curl bind-utils net-tools ipset libtool-ltdl rsync nfs-utils
```

Comment:
@lloesche and I had a chat about this and we agreed that making the `nfs-utils` package available in our default centos image will make it easier for anyone trying to migrate legacy applications or run Jenkins (most of our Jenkins users are backing their storage by NFS). I would like to continue to encourage and educate users about the limitations with NFS as central storage for a DC/OS cluster, but at the end of the day I think it's ok to let them make this judgement call. Also they would still have to set up the NFS mounts on their cluster so this won't be something they start using without some explicit action (and hopefully review of EFS limitations) on their part.